huggingface/base/inference/artifacts/mms-entrypoint.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I would this file place in `build_artifacts` instead of `artifacts` to keep it the same across all frameworks.
https://github.com/vdantu/deep-learning-containers/tree/hugging_face/pytorch/inference/docker/build_artifacts

Reply:
There are framework repositories with "artifacts" and "build-artifacts" .. We could move it to either, I don't have any preference.